### ISSUE TITLE
Trakt.TV / aeonlucid.com Lists Down?

### ISSUE THREAD

**Issue by rkmfl** [Radarr/Radarr#1617]
**Description:**
I searched and did not find any post with this problem, making me thing it's only on my side.  However, I've tested everything I can find .. on two different machines and have the exact same issue.

The issue is that Radarr routes the "list" request for Trakt.tv through "aeonlucid.com" which appears to be down.

Any time Radarr attempts to get a Trakt.tv list update, it receives an error:

> HTTP Error - Res: [GET] http://radarr.aeonlucid.com/v1/trakt/refresh?refresh=REDACTEDforPOST: 500.InternalServerError

(**NOTE:** I've replaced my personal GUID with "REDACTEDforPOST")

**Radarr Version:**
0.2.0.696

**Logs:**
> 17-6-4 00:01:13.9|Info|RssSyncService|Starting RSS Sync
> 17-6-4 00:01:15.8|Warn|HttpClient|HTTP Error - Res: [GET] http://radarr.aeonlucid.com/v1/trakt/refresh?refresh=REDACTEDforPOST: 500.InternalServerError
> 17-6-4 00:01:15.8|Warn|TraktImport|Trakt Top By Week List HTTP request failed: [500:InternalServerError] [GET] at [http://radarr.aeonlucid.com/v1/trakt/refresh?refresh=REDACTEDforPOST]
> 17-6-4 00:01:19.5|Info|DownloadDecisionMaker|Processing 300 releases
> 17-6-4 00:01:26.1|Info|RssSyncService|RSS Sync Completed. Reports found: 300, Reports grabbed: 0
> 17-6-4 00:01:48.7|Warn|HttpClient|HTTP Error - Res: [GET] http://radarr.aeonlucid.com/v1/trakt/refresh?refresh=REDACTEDforPOST: 500.InternalServerError
> 17-6-4 00:01:48.7|Warn|TraktImport|Unable to connect to list [v0.2.0.696] NzbDrone.Common.Http.HttpException: HTTP request failed: [500:InternalServerError] [GET] at [http://radarr.aeonlucid.com/v1/trakt/refresh?refresh=REDACTEDforPOST]
   at NzbDrone.Common.Http.HttpClient.Execute(HttpRequest request) in C:\projects\radarr-usby1\src\NzbDrone.Common\Http\HttpClient.cs:line 99
   at NzbDrone.Core.NetImport.Trakt.TraktRequestGenerator.Authenticate() in C:\projects\radarr-usby1\src\NzbDrone.Core\NetImport\Trakt\TraktRequestGenerator.cs:line 66
   at NzbDrone.Core.NetImport.Trakt.TraktRequestGenerator.<GetMovies>d__16.MoveNext() in C:\projects\radarr-usby1\src\NzbDrone.Core\NetImport\Trakt\TraktRequestGenerator.cs:line 132
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at NzbDrone.Core.NetImport.HttpNetImportBase`1.TestConnection() in C:\projects\radarr-usby1\src\NzbDrone.Core\NetImport\HttpNetImportBase.cs:line 144

**Comment by galli-leo** [comment]
We don't route the list request through aeonlucid.com, only to get your auth token from trakt.tv

**Comment by rkmfl** [comment]
@galli-leo Thank you for the clarification.

In either case, the net result is that aeonlucid.com has not been responding for some time and nothing can be done from the end-user's side as a workaround, is that correct?

**Comment by galli-leo** [comment]
Correct, if you have private lists. You can make them public and it should be fine for radarr.

**Comment by rkmfl** [comment]
Unfortunately, I'm experiencing the issue **only with public lists**.

I'm choosing from the pre-populated list provided by Radarr, for example "**Trending**" or "**TopWatchedByWeek**".

Both of those public lists have triggered the error I've included in my logs above.

**Comment by galli-leo** [comment]
Gotcha. In that case try "signing out of trakt" in Radarr. Then we shouldn't try to authenticate anymore.

**Comment by AeonLucid** [comment]
Investigating..

**Comment by galli-leo** [comment]
@rkmfl Can you try reseting your trakt tokens inside radarr settings?

**Comment by AeonLucid** [comment]
Pushed an update (https://github.com/Radarr/RadarrAPI.Update/commit/b43af18f7345f1919a709c5c1b2502e6c886dc08) to the API which makes it possible to handle the error in Radarr.

**Comment by rkmfl** [comment]
@galli-leo - 
You are correct, Radarr was off for about 3 weeks.  I disabled the service once it ran into that problem where it downloaded the sew few movies hundreds of times and 400GB worth.  Having not run Radarr for 3 weeks, must be the cause of the problem because, I did not revoke anything in Trakt.

I had previously tried revoking the token in Radarr; however, once you requested it here, I did it again and realized that the first time I had done it wrong.  

Radarr pops a message that the user must Save the changes after revoking, and I missed that the prior time.  In short, you are correct that properly revoking the Trakt.tv token does fix this issue, thank you.

I probably good that I ran into this, because it's probably a good idea to have some error message shown to users in the app upon this issue, based on your thoughts.

I was also thinking the reason it may have been routed through **AeonLucid**, is due to my having entered by **username** in the username field of the Trakt.tv list.  Although I was using a public list, perhaps simply having completed this field causes Trakt to use the **AeonLucid** service.  I was not aware of that originally, and just doing my best to complete all fields when I setup the list,

I wonder if it might be a good idea to add a note to the help tooltip on the username field, or even a sub-head next to the title that states "**not required when using a public Trakt list**"

Thanks again for investigating this, and hopefully anyone else that run into the issue will find this thread in the future.
